docs/index.md

```diff
@@ -750,6 +753,183 @@ signature | string | *Optional* Signed claim authorizing withdrawal of XRP from
 ```
 
 
+# rippled APIs
+
+ripple-lib relies on [rippled APIs](https://ripple.com/build/rippled-apis/) for all online functionality. With ripple-lib version 0.22.0 and higher, you can easily access rippled APIs through ripple-lib. Use the `request()`, `hasNextPage()`, and `requestNextPage()` methods:
```

Comment:
Don't worry about the indicated version number, I will update this to be accurate before the release goes out.